advanced/events.md

```diff
@@ -392,16 +394,16 @@ tinymce.on('AddEditor', function (e) {
 | Event       |  Description   |
 |-------------|----------------|
 | [AddEditor](#addeditor) | Fires when a new editor instance is added. |
-| [RemoveEditor](#removeeditor) | Fires when a editor instance is removed. |
+| [RemoveEditor](#removeeditor) | Fires when an editor instance is removed. |
 
 ### AddEditor
 
-This event gets fired when editor instances gets created and added to the EditorManager collection.
+This event gets fired when editor instances get created and added to the EditorManager collection.
```

Comment:
This event is fired when an editor instance is created and added to the EditorManager collection.

Reply:
Changed as suggested by @techtangents .